pkg/log/log_test.go

```diff
@@ -0,0 +1,36 @@
+package log_test
+
+import (
+	"io/ioutil"
+	"os"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/weaveworks/pctl/pkg/log"
+)
+
+var _ = Describe("PrintLogger", func() {
+	var (
+		r   *os.File
+		w   *os.File
+		tmp *os.File
+	)
+
+	BeforeEach(func() {
+		r, w, _ = os.Pipe()
+		tmp = os.Stdout
+		os.Stdout = w
+	})
+
+	It("logs a message to stdout with a tickmark", func() {
+		defer func() {
+			os.Stdout = tmp
+		}()
+
+		log.Warningf("test-warning")
```

Comment:
Only testing one of the log funcs because they're all `format()` and only the tickmark changes.